bindings/nodejs/types/account.ts

```diff
@@ -12,30 +13,39 @@ export interface AccountBalance {
     total: string;
     available: string;
     requiredStorageDeposit: string;
-    nativeTokens: nativeTokensMap;
+    nativeTokens: INativeToken[];
     nfts: string[];
     aliases: string[];
     foundries: string[];
-    potentiallyLockedOutputs: potentiallyLockedOutputsMap;
+    /**
+     * Outputs with multiple unlock conditions and if they can currently be spent or not. If there is a
+     * TimelockUnlockCondition or ExpirationUnlockCondition this can change at any time
+     */
+    potentiallyLockedOutputs: { [outputId: string]: boolean };
 }
 
-export type nativeTokensMap = {
-    [tokenId: string]: number;
-};
-
-export type potentiallyLockedOutputsMap = {
-    [outputId: string]: boolean;
-};
-
 export interface AccountSyncOptions {
+    /**
+     * Specific Bech32 encoded addresses of the account to sync, if addresses are provided,
+     * then `address_start_index` will be ignored
+     */
     addresses?: string[];
+    /**
+     * Address index from which to start syncing addresses. 0 by default, using a higher index will be faster because
+     * addresses with a lower index will be skipped, but could result in a wrong balance for that reason
+     */
     addressStartIndex?: number;
-    automaticOutputConsolidation?: boolean;
+    /**
+     * Usually we skip syncing if it's called within a few seconds, because there can only be new changes every 10
```

Comment:
Milestone interval changed to 5 seconds (the comment in Rust code also needs to be updated)

Reply:
I will fix both.